### ISSUE TITLE
iOS app not exists on Fabric Dashboard

### ISSUE THREAD

**Issue by birfincankafein** [sarriaroman/FabricPlugin#130]
I added fabric as you mentioned in the guide. Android working perfect but iOS not working. Even I see XXX (Android) in the dashboard but iOS not exists.

**Comment by AbuHani** [comment]
i have same issue, did you solve it ?

**Comment by birfincankafein** [comment]
No, still waiting and I checked the device's logs, it looks perfect, no error is given.

**Comment by AbuHani** [comment]
I created new organization and added the APP with new API KEYS
